main/utils.py

```
@@ -778,11 +778,7 @@ def update_tenant_quota_for_max_allowed_sku(request: HttpRequest, skus: str) ->
         user_quota.save()
     else:
         raise QuotaExceededException(
-            message=(
-                "Превышен лимит количества товаров для данного тарифа. %s / %s",
-                skus_count,
-                user_quota.skus_limit,
-            ),
+            message=(f"Превышен лимит количества товаров для данного тарифа."),
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)